File: exercises/java/src/test/java/tax/DummyTaxCalculator.java
package tax;

// DO NOT MAKE ANY CHANGES TO THIS CLASS - IT IS JUST HERE TO KEEP THE TESTS HAPPY TO BEGIN WITH
// DON'T BE LAZY - WRITE YOUR OWN NEW TAXCALCULATOR CLASS
@Deprecated
public final class DummyTaxCalculator extends TaxCalculator {
    @Override
    public int calculateTax(Vehicle vehicle) {
        return -1;
    }
}

File: exercises/dotnet/TaxCalculator.Tests/DummyTaxCalculator.cs
using System;

namespace TaxCalculator.Tests
{
    [Obsolete("DO NOT MAKE ANY CHANGES TO THIS CLASS - IT IS JUST HERE TO KEEP THE TESTS HAPPY TO BEGIN WITH. DON'T BE LAZY - WRITE YOUR OWN NEW TAXCALCULATOR CLASS")]
    public sealed class DummyTaxCalculator : TaxCalculator
    {
        public override int CalculateTax(Vehicle vehicle)
        {
            return -1;
        }
    }
}
